DfE.FindInformationAcademiesTrusts/Pages/Cookies.cshtml

```diff
@@ -0,0 +1,140 @@
+        </tr>
+        <tr>
+          <td class="govuk-table__cell">ASLBSA </td>
+          <td class="govuk-table__cell">Used to balance the load on our servers </td>
+          <td class="govuk-table__cell">With browser settings (Session)</td>
+        </tr>
+        <tr>
+          <td class="govuk-table__cell">ASLBSACORS </td>
+          <td class="govuk-table__cell">Used to balance the load on our servers </td>
+          <td class="govuk-table__cell">With browser settings (Session)</td>
+        </tr>
+        <tr>
+          <td class="govuk-table__cell">.AspNetCore.Antiforgery </td>
+          <td class="govuk-table__cell">Secures you and our service against Cross Site Scripting Attacks (CSRF) </td>
+          <td class="govuk-table__cell">With browser settings (Session)</td>
+        </tr>
+      </tbody>
+    </table>
+
+    <h2 class="govuk-heading-m">Optional analytics cookies</h2>
+    <p class="govuk-body">
+      With your permission, we use Application Insights to collect data about how you use this site. This information helps us to improve our service.
+    </p>
+    <p class="govuk-body">
+      Application Insights is not allowed to use or share our analytics data with anyone. Application Insights stores anonymised information about:
+    </p>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>how you got to this site</li>
+      <li>the pages you visit on this site and how long you spend on them</li>
+      <li>any errors you see while using this site</li>
+      <li>what you click on while you are visiting the site</li>
+    </ul>
+
+    <table class="govuk-table" aria-label="Essential cookies">
+      <thead class="govuk-table__header">
+        <tr>
+          <th class="govuk-table__header">Name</th>
+          <th class="govuk-table__header">Purpose</th>
+          <th class="govuk-table__header">Expires</th>
+        </tr>
+      </thead>
+      <tbody class="govuk-table__body">
+        <tr>
+          <td class="govuk-table__cell">ai_user</td>
+          <td class="govuk-table__cell">Checks if you have visited this site before. This helps us count how many people visit our site.</td>
+          <td class="govuk-table__cell">1 year</td>
+        </tr>
+        <tr>
+          <td class="govuk-table__cell">ai_session</td>
+          <td class="govuk-table__cell">Checks if you have visited this site before. This helps us count how many people visit our site.</td>
+          <td class="govuk-table__cell">30 minutes</td>
+        </tr>
+      </tbody>
+    </table>
+  </div>
+</div>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <form method="post" name="frmCookies">
+      <div class="govuk-form-group">
+        <input type="hidden" name="redirectPath" value="@(HttpContext.Request.Path + HttpContext.Request.QueryString)"/>
+        <fieldset class="govuk-fieldset">
+          <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
```

Comment:
```suggestion
          <legend class="govuk-fieldset__legend govuk-fieldset__legend--m">
```
This text shouldn't be the same size as the page header - reducing it to medium fixes this